it/src/main/java/org/corfudb/universe/node/CorfuServer.java

```diff
@@ -0,0 +1,71 @@
+package org.corfudb.universe.node;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import org.corfudb.runtime.view.Layout;
+import org.slf4j.event.Level;
+
+import java.time.Duration;
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Optional'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)